docs/epics-and-tasks.md

```
@@ -1,186 +1,141 @@
-2️⃣ Повний список EPIC → TASK (High-Level Roadmap)
+# Dota 2 Analytics Pipeline — Roadmap
 
-Це твій технічний план проєкту, який можна винести в:
+## Легенда статусів
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)